From 8ebab1805deba8913ef1941a1f5d51df58b01208 Mon Sep 17 00:00:00 2001
From: Steven Morgan <stevmorg@cisco.com>
Date: Thu, 5 Jan 2017 12:30:35 -0500
Subject: bb111711 - fix zlib version check - patch by Daniel J. Luke.

Patch-Name: bb111711-fix-zlib-version-check-patch-by-Daniel-J.-L.patch
---
 m4/reorganization/libs/libz.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/m4/reorganization/libs/libz.m4 b/m4/reorganization/libs/libz.m4
index b5c7414..f7b67ca 100644
--- a/m4/reorganization/libs/libz.m4
+++ b/m4/reorganization/libs/libz.m4
@@ -29,9 +29,9 @@ then
     AC_MSG_ERROR([Please install zlib and zlib-devel packages])
 else
 
-    vuln=`grep "ZLIB_VERSION \"1.2.0" $ZLIB_HOME/include/zlib.h`
+    vuln=`grep "ZLIB_VERSION \"1.2.0\"" $ZLIB_HOME/include/zlib.h`
     if test -z "$vuln"; then
-	vuln=`grep "ZLIB_VERSION \"1.2.1" $ZLIB_HOME/include/zlib.h`
+	vuln=`grep "ZLIB_VERSION \"1.2.1\"" $ZLIB_HOME/include/zlib.h`
     fi
 
     if test -n "$vuln"; then
